### PR TITLE
added new MD file for mapping tutorials

### PR DESCRIPTION
**Pull request by twinblade02**
requesting review

### CODE REVIEW

File: tutorials-fall-2022/python/tutorial-7-geoMaps.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+countyMaps.drop(['countyns', 'namelsad', 'stusab', 'lsad', 'classfp', 'mtfcc',
+                 'csafp', 'cbsafp', 'metdivfp', 'funcstat', 'aland', 'aland',
+                 'awater', 'state_name', 'countyfp_nozero'], axis = 1, inplace=True)
+
+countyMaps['countyfips'] = countyMaps['countyfips'].astype(np.int64)
+countyMaps.drop(['geometry'], axis=1, inplace=True)
+merged_df = pd.merge(<your_dataframe>, countyMaps, on='countyfips', how='inner')
+```
+
+The code above is simply preprocessing the files you need to get them to a point where it is possbile to visualize data with Folium quickly.
```
Comment (harshpavuluri) [inline tutorials-fall-2022/python/tutorial-7-geoMaps.md]: "The code above is simply preprocessing the files. You will need to additionally process them in a way that you see fit to visualize them with Folium, which we will show you how to use later on."

Editing for clarity here

**Review by harshpavuluri — CHANGES_REQUESTED** [review]
Looks good for the most part. Just that 1 change. Also do a spell check real quick as well as I did see some spelling errors on that 1 line I commented

**Review by twinblade02 — COMMENTED** [review]
Will do. I'll check the grammar.

**Review by harshpavuluri — APPROVED** [review]
LGTM

**Pull request merged**